openshift/4.0/templates/api/build.yaml

```diff
@@ -50,53 +50,22 @@ parameters:
     displayName: Context Directory
     description: Set this to use a subdirectory of the source code repository
     value: backend
-
-  - name: DOTNET_BUILDER_IMAGE
-    displayName: .NET builder
-    description: The s2i image which is used to build the code.
-    required: true
-    value: dotnet-31-rhel7
-  - name: DOTNET_BUILDER_TAG
-    displayName: .NET builder tag
-    description: The s2i image tag which is used to build the code.
-    required: true
-    value: "3.1"
-
-  - name: DOTNET_STARTUP_PROJECT
```

Comment:
All of this was specific to the original RedHat image.  It is no longer required.